### ISSUE TITLE
ASP.NET Core RC2 Web API Application

### ISSUE THREAD

**Issue by rustd** [OmniSharp/generator-aspnet#692]
This is a close to final sample for RC2. Attaching the sample below
[WebAPI.zip](https://github.com/OmniSharp/generator-aspnet/files/260048/WebAPI.zip)

.NET CLI
 .NET Command Line Tools (1.0.0-rc2-002673) or higher
 Install Location: https://github.com/dotnet/cli#installers-and-binaries

NuGet Feed
https://www.myget.org/F/aspnetrelease/api/v3/index.json
https://api.nuget.org/v3/index.json

//cc @sayedihashimi

**Comment by peterblazejewicz** [comment]
This one outputs warnings during build:
```
dotnet build
Project WebAPIApplication (.NETCoreApp,Version=v1.0) will be compiled because expected outputs are missing
Compiling WebAPIApplication for .NETCoreApp,Version=v1.0
/Users/piotrblazejewicz/development/WebAPIApplication/project.json(7,50): warning NU1007: Dependency specified was Microsoft.AspNetCore.Mvc >= 1.0.0-rc2-20801 but ended up with Microsoft.AspNetCore.Mvc 1.0.0-rc2-20825.
/Users/piotrblazejewicz/development/WebAPIApplication/project.json(10,81): warning NU1007: Dependency specified was Microsoft.Extensions.Configuration.EnvironmentVariables >= 1.0.0-rc2-20801 but ended up with Microsoft.Extensions.Configuration.EnvironmentVariables 1.0.0-rc2-20825.
/Users/piotrblazejewicz/development/WebAPIApplication/project.json(11,75): warning NU1007: Dependency specified was Microsoft.Extensions.Configuration.FileExtensions >= 1.0.0-rc2-20801 but ended up with Microsoft.Extensions.Configuration.FileExtensions 1.0.0-rc2-20825.
/Users/piotrblazejewicz/development/WebAPIApplication/project.json(12,65): warning NU1007: Dependency specified was Microsoft.Extensions.Configuration.Json >= 1.0.0-rc2-20801 but ended up with Microsoft.Extensions.Configuration.Json 1.0.0-rc2-20825.
/Users/piotrblazejewicz/development/WebAPIApplication/project.json(14,62): warning NU1007: Dependency specified was Microsoft.Extensions.Logging.Console >= 1.0.0-rc2-20801 but ended up with Microsoft.Extensions.Logging.Console 1.0.0-rc2-20825.
/Users/piotrblazejewicz/development/WebAPIApplication/project.json(15,60): warning NU1007: Dependency specified was Microsoft.Extensions.Logging.Debug >= 1.0.0-rc2-20801 but ended up with Microsoft.Extensions.Logging.Debug 1.0.0-rc2-20825.
```
Except of this, everything's nice&dandy

**Comment by peterblazejewicz** [comment]
closed via: 3b28476
